db/migrate/20240322214921_index_adopter_applications_on_account_and_pet.rb

```diff
@@ -0,0 +1,7 @@
+class IndexAdopterApplicationsOnAccountAndPet < ActiveRecord::Migration[7.0]
+  def change
+    add_index :adopter_applications, [:pet_id, :adopter_foster_account_id],
+      unique: true,
+      name: "index_adopter_applications_on_account_and_pet"
```

Comment:
I had to manually name the index because, with the new adopter_foster_account name, this default index name was getting too long.